compose.yaml

```diff
@@ -12,7 +12,9 @@ services:
       retries: 300
     env_file:
       - env.d/development/postgresql
-      - env.d/development/postgresql.local
+      - env.d/development/postgresql.${ENV_OVERRIDE:-local}
+    volumes:
+      - ./data/postgresql.${ENV_OVERRIDE:-local}:/var/lib/postgresql/data
```

Comment:
Why do you need this ?

Reply:
Because `POSTGRES_DB` is used only the first time postgresql container is started

Reply:
That means if we change POSTGRES_DB on subsequent run it would still use the previous one. By switching volume it really switch the DB.

Reply:
And if the variables change, it is not recreated ?